nais/prod-fss.yml

```diff
@@ -90,14 +90,8 @@ spec:
       value: api://prod-fss.pdl.pdl-api/.default
 
```

Comment:
Det mangler outbound-rules for k9-sak

Reply:
Trenger ikke det i FSS. Inbound brukes kun for å gi tilgang til å hente token.
Med det sagt skulle jag fjernet den i dev-fss :)

Reply:
Aha OK 👍